grails-app/views/organization/list.gsp

```diff
@@ -15,7 +15,7 @@
             <div class="button-bar">
                 <g:link class="button" action="list">
                     <img src="${resource(dir: 'images/icons/silk', file: 'application_side_list.png')}" />&nbsp;
-                    <warehouse:message code="default.list.label" args="['organization']"/>
+                    <warehouse:message code="default.list.label" args="['organizations']"/>
```

Comment:
Let's use args="[g.message(code:'organizations.label')]" instead of args="['organizations']"'.